docker-compose.yaml

```diff
@@ -30,3 +32,36 @@ services:
     ports:
       - 5971:5971
       - 5972:5972 
+  cardano-node:
+    image: ghcr.io/intersectmbo/cardano-node:9.2.0
+    command: [
+      "run",
+      "--config", "/config/config.json",
+      "--database-path", "/data/db",
+      "--socket-path", "/ipc/node.socket",
+      "--topology", "/config/topology.json"
+    ]
+    volumes:
+      - ./server/config/network/${NETWORK:-preprod}/cardano-node:/config
+      - ./server/config/network/${NETWORK:-preprod}/genesis:/genesis
+      - node-db:/data
+      - node-ipc:/ipc
+    restart: on-failure
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "400k"
+        max-file: "20"
+  ogmios:
+    image: cardanosolutions/ogmios:latest
```

Comment:
We can explicitly set the version here instead of `latest` - just to make sure it always matches `9.2.0`

Reply:
I see the biggest version is 6.8.0 [here](https://hub.docker.com/r/cardanosolutions/ogmios/tags)
I also check Compatibility [here](https://github.com/CardanoSolutions/ogmios): They only mention for 6.7.* will match with cardano-node 9.2.0
=> So I prefer 6.7.1 for ogmios image: cardanosolutions/ogmios:6.7.1

Reply:
Hmm https://github.com/CardanoSolutions/ogmios/releases - looks like `6.8.0` matches `9.2.0`

Reply:
Yep, Let's use 6.8.0